.github/workflows/pr.yml

```diff
@@ -186,7 +222,7 @@ jobs:
 
   cli:
     name: CLI
-    runs-on: [runs-on, runner=8cpu-linux-x64, "run-id=${{ github.run_id }}"]
+    runs-on: [runs-on, runner=8cpu-linux-x64, disk=medium, "run-id=${{ github.run_id }}"]
```

Comment:
just curious what happened here

Reply:
checkout the current failing CI, its out of space on CLI job, maybe we need large here too?